icons/square/48/terminus.svg

```diff
@@ -0,0 +1,20 @@
+<svg width="48" height="48" version="1.1" viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Why there's a width & height property while on the circle one there isn't? I don't know if we should keep them as a lot of other icons have both. As far as I know, viewBox only is enough :) maybe the missing width & height can cause issues with Qt or another SVG rendering library